### DISCUSSION TITLE
Welcome to homebridge-airport-express-playing Discussions!

### DISCUSSION THREAD

**Discussion by apexad** [apexad/homebridge-airport-express-playing#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lucanaut** [comment]
Hello,
Is this plug in for specific models of the Airport Express?  I have the older kind (not the one that looks like a white Apple TV) but I don't see it appear on my home app.  Maybe it's for the newer one?  
Thank you.

**Reply by apexad** [reply]
Correct, it only works with the final Airport Express that supports Airplay 2.

https://support.apple.com/kb/SP651?locale=en_US
